app/src/main/assets/InstagramResponseExample.json

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
What is this? Needed to mock something ?? or do you need this for any connection with instagram?

Reply:
No, is only to show an example of api success response (because there are many fields in the response).
I don't know if needs to be deleted